lib/init.js

```diff
@@ -59,101 +72,22 @@ async function getLatestRelease(owner, repo) {
 }
 
 module.exports = async () => {
-  logger.info("------");
-  logger.info("");
-  logger.info(theme.green("Welcome to the Testdriver Setup!"));
-  logger.info("");
-  logger.info("Choose a Runner");
-  logger.info("│");
-  logger.info("├── Local Runner:" + theme.green(" Free"));
-  logger.info("│    └── " + theme.dim("Run tests on your computer"));
-  logger.info("│");
-  logger.info(
-    "└── Sandbox Runner:" + theme.green(" 7 Day Trial then $20/month"),
-  );
-  logger.info(
-    "     └── " + theme.dim("Run tests on private hosted ephemeral VMs"),
-  );
-  logger.info("         ├── ✅ " + "Added Privacy");
-  logger.info("         ├── ✅ " + "Faster Execution");
-  logger.info("         ├── ✅ " + "Parallel Execution");
-  logger.info("         └── ✅ " + "Better DX");
-  logger.info("");
+  // Parse CLI args
+  const cliArgs = parseArgs();
+
+  // Map CLI args to expected keys with defaults
+  const response = {
```

Comment:
No need for these, since they're actually `.env` variables!

```bash
TD_API_KEY=123 TD_WEBSITE=something npx testdriver@latest
```

So the logic can do something like:

```ts
if (!process.env.TD_API_KEY) {
  throw new Error('Missing TD_API_KEY')
}
```

Reply:
I see that the `init --...` goal is to populate the `.env` on behalf of the user.

I think it's fair to expect to encourage convention like `.env` or passing in variables via shell.

There's some precedent for supporting _both_ (e.g. `-p` overriding `PORT` https://nextjs.org/docs/app/api-reference/cli/next#changing-the-default-port), but I believe we should try to prevent as much supported surface area where possible.  (Including behavior like "TD_API_KEY is in the .env, _and_ the environment already, but also passed as a flag: which one wins?")

Relying on `process.env.*` also means we can get 1 step closer to not needing to `init` at all 🙏